modules/color.js

```diff
@@ -0,0 +1,47 @@
+const { INTEGER } = require('./number');
+const rndNum = (min = 0, max = 255) => INTEGER(min, max); //Math.round(Math.random() * 255);
```

Comment:
There is no need to shorten names of constants, in this case; we should keep it readable as possible, so, could you just find and replace all `rndNum` to `randomNumber`?

And one more thing, is there a reason why that line is not like this bellow?
`const randomNumber = INTEGER(0, 255)`

Reply:
Random color generation has changed so that this is no longer needed.